docs/operations.md

```diff
@@ -18,10 +18,54 @@ then deleting the old chunks. The scavenged events are also removed from the ind
 
 Running a scavenge causes the active chunk to be completed so that it can be scavenged.
 
-::: warning 
+::: warning
 Scavenging is destructive. Once a scavenge has run, you cannot recover any deleted events except from a backup.
 :::
 
+### Mechanism of the scavenging
+
+Running scavenge for the first time will take longer than usual because it needs to create a "scavenge point", then execute all phases, create folders and SQL database files, and finally mark the current chunk as completed before removing it.
```

Comment:
Running scavenge for the first time will take longer than usual because it needs to create a "scavenge point", then execute all phases, create folders and SQL database files, and finally mark the current chunk as completed before removing it.

->

Running scavenge for the first time will take longer than subsequent scavenges because it needs to create a "scavenge point" as part of the first phase i.e. accumulation phase. This is followed by two more phases at the end of which events are removed from the database, including from the current chunk. More information on the phases can be found below.